src/terra-core/TerraCoreDataModel.ttl

```diff
@@ -1565,7 +1563,6 @@ TerraCore:hasAlpha2Code
 .
 TerraCore:hasAmount
   a owl:DatatypeProperty ;
-  rdfs:comment "Can replace this with a measurment ontology" ;
```

Comment:
Suggest: "Consider replacing with a measurement ontology" -- spelling and more specific.